### DISCUSSION TITLE
any functions can be used to choose dimToUse ?

### DISCUSSION THREAD

**Discussion by yingyuan830** [GreenleafLab/ArchR#1538] [Questions / Documentation]
There are many functions need to set the dimToUse parameter, but I didn't find any functions can be used to choose a proper dimToUse parameter, can anyone give hints or other suggestions ? Thanks

**Comment by rcorces** [comment]
ArchR automatically excludes dims that are highly correlated with sequencing depth (`corCutOff`). We leave it up to the user to determine what other dims they might want to exclude based on their own intuition of their data.

**Reply by yingyuan830** [reply]
Thanks, but I am still confused about these, could you please give more explain ? 
1. the dimToUse can be used to choose the dims from the dims after corCutOff filter or just from the whole dims after dimension reduction ?
2. how to make a better setting for corCutOff ? why 0.75 can be a general value ?

**Reply by rcorces** [reply]
1. dimsToUse would filter the dims separately and in addition to corCutOff. If corCutOff removes dim1 and you tell dimsToUse to not use dim2, you would remove both dim1 and dim2.
2. As with all ArchR parameters, we have done our best to select broadly applicable default values. But it is up to the end user to understand those parameters and decide if they apply to their own data or how they should be altered for their own data.

**Reply by yingyuan830** [reply]
ok, thank you
